pages.de/common/nginx.md

```diff
@@ -0,0 +1,24 @@
+# nginx
+
+> Nginx Webserver.
+> Weitere Informationen: <https://nginx.org/en/>.
+
+- Starte den Server mit der standardmäßigen Konfigurationsdatei:
+
+`nginx`
+
+- Starte den Server mit einer benutzerdefinierten Konfigurationsdatei:
+
+`nginx -c {{konfigurationsdatei}}`
```

Comment:
```suggestion
`nginx -c {{konfiguration_datei}}`
```
Would this be better?

Reply:
Thanks for your suggestion. In German we can combine words by appending them to each other. The word `Konfigurationsdatei` is the established [translation of a config file](https://translate.google.com/?sl=en&tl=de&text=config%20file&op=translate) and is part of the official [Duden](https://www.duden.de/rechtschreibung/Konfigurationsdatei) directory. If you wish, we could change it to `pfad/zu/konfigurationsdatei`.